packages/cli/src/commands/console.js

```diff
@@ -9,19 +10,51 @@ export const command = 'console'
 export const aliases = ['c']
 export const description = 'Launch an interactive Redwood shell (experimental)'
 
-const paths = getPaths().api
+const paths = getPaths()
 
-const mapDBToContext = (ctx) => {
-  const { db } = require(path.join(paths.lib, 'db'))
-  ctx.db = db
+const loadPrismaClient = (replContext) => {
+  const { db } = require(path.join(paths.api.lib, 'db'))
+  replContext.db = db
+}
+
+const loadUserConfig = (replContext) => {
```

Comment:
In the PR description, you mention `api/src/config/console.[ts|js]`. I was going to ask about using `lib/` instead of `config/`, which I see you are using here. All good if so! Just reconciling the two things.

Instead of adding the file to the Create RW App template (not sure whether or not that what you are planning), what if this try/catch created the file if it doesn't exist?
- console message to the user the file is created; link to doc how to use
- check for language target using `getProject().isTypeScriptProject` from Structure package
- and/or could create a new CLI command to create this file from a template (which we do for `tsconfig.json` as a `rw setup ...` command)

Reply:
I think there might be some confusion, but the `api/src/config/console` path is the one the command looks for. The `lib/` path I mentioned was just an example of the sort of project-relative import that is currently broken.

I didn't have concrete plans to create this file in the generator, but adding a creation step in the `catch` statement seems reasonable. I guess it comes down to whether you think most people who use the REPL will want to customize its behavior/set custom variables.

If we *do* end up autogenerating the user config file, then I'd probably move the binding of `db` into that file as well to show the user how that works and avoid the console using more magic than necessary.

Reply:
Gah, you're correct. I confused the paths between db and console when I was (quickly) reviewing. `config` would be the "right" directory. Just thinking off the top of my head if there was a simplicity factor to consider... Regardless, ignore and move ahead!  

> I guess it comes down to whether you think most people who use the REPL will want to customize its behavior/set custom variables.

^^ I'd need to think on use-cases here. Dom might have more informed opinions.

> If we do end up autogenerating the user config file, then I'd probably move the binding of db into that file as well to show the user how that works and avoid the console using more magic than necessary.

^^ +1 to this for sure. And feels like a great reason to create the file.